airbyte-oauth/src/main/java/io/airbyte/oauth/flows/GithubOAuthFlow.java

```diff
@@ -30,7 +32,18 @@
   // tune up.
   // This is necessary to pull data from private repositories.
   // https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes
-  private static final String SCOPES = "repo";
+
+  private static final List<String> SCOPES = Arrays.asList(
+      "repo",
+      "read:org",
+      "read:repo_hook",
+      "read:user",
+      "read:discussion",
+      "workflow");
+
+  public String getScopes() {
+    return String.join("%20", SCOPES);
+  }
```

Comment:
"%20" is the code " " (space)
Try to concatenate this array by simple space and use the standard URIBuilder for generation of links

Reply:
By using default UriBuilder flow, the others symbols like ` : ` are also encoded. To grant the permissions correctly, we should keep this method of encoding, like implemented. 
Also, refer to this article about oauth scopes:  https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps (Note section, after scopes table)